module-cheatsheets/scRNA-seq-advanced-cheatsheet.md

```diff
@@ -56,22 +57,33 @@ Read the [`miQC` package documentation](https://rdrr.io/github/greenelab/miQC/),
 
 
 | Library/Package      | Piece of Code      | What it's called    | What it does  |
-|----------------------|----------------------------|--------------------------------------------|--------------------------------------------------------------|
+|----------------------|--------------------|---------------------|---------------|
 | `miQC`| [`mixtureModel()`](https://rdrr.io/github/greenelab/miQC/man/mixtureModel.html)| Mixture model | Fit a `miQC` mixture model to a `SingleCellExperiment` object for use in filtering |
 | `miQC`| [`filterCells()`](https://rdrr.io/github/greenelab/miQC/man/filterCells.html)| Filter cells | Filter cells from a `SingleCellExperiment` object based on a `miQC` model, returning a filtered `SingleCellExperiment` object |
 | `miQC`| [`plotMetrics()`](https://rdrr.io/github/greenelab/miQC/man/plotMetrics.html)| Plot metrics | Plot percent of mitochondrial reads against the number of unique genes found for each cell |
 | `miQC`| [`plotModel()`](https://rdrr.io/github/greenelab/miQC/man/plotModel.html)| Plot model | `miQC::plotMetics()` with the `miQC` fitted model overlaid |
 | `miQC`| [`plotFiltering()`](https://rdrr.io/github/greenelab/miQC/man/plotFiltering.html)| Plot filtering | Plot percent of mitochondrial reads against the number of unique genes found, coloring points based on whether they will be filtered out or not |
 
 
+## `SingleR`
+
+Read the [`SingleR` package documentation](https://rdrr.io/bioc/SingleR/), and a [e-book on its usage](http://bioconductor.org/books/release/SingleRBook/).
+
+| Library/Package      | Piece of Code      | What it's called    | What it does  |
+|----------------------|--------------------|---------------------|---------------|
+| `SingleR` | [`SingleR()`](https://rdrr.io/bioc/SingleR/man/SingleR.html) | Annotate scRNA-seq data | Combines the below "training" and classification functions into a single function to assign cell types to the cells in an SCE object using an annotated reference dataset |
```

Comment:
```suggestion
| `SingleR` | [`SingleR()`](https://rdrr.io/bioc/SingleR/man/SingleR.html) | Annotate scRNA-seq data | Combines the below "training" and classification functions into a single function to assign cell types to the cells in an `SCE` object using an annotated reference dataset |
```

Reply:
If the long-ish description makes the PDF render look bad, we could move it to be the last item in the last and change the description to "convenience function to run the previous two functions in one call" or something similarly quick.